app/uk/gov/hmrc/customs/rosmfrontend/domain/TaxEnrolmentsResponse.scala

```diff
@@ -18,13 +18,22 @@ package uk.gov.hmrc.customs.rosmfrontend.domain
 
 import org.joda.time.format.DateTimeFormat
 import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.customs.rosmfrontend.models.enrolmentRequest.KeyValuePair
 
 case class Identifier(key: String, value: String)
 
 object Identifier {
   implicit val jsonFormat = Json.format[Identifier]
 }
 
+case class Verifier(key: String, value: String)
```

Comment:
Not sure if this should live here, because the whole class is TaxEnrolmentsResponse.

If preferable I can create separate Identifier model and this Verifier model and put this in the Request model. What do you think @SteveSugden ?

Reply:
I think I agree.  If Verifier is nothing to do with TaxEnrolmentResponse then move it.   I prefer classes in their own files normally but I'm old-fashioned.

Reply:
Extracted to separate files. I'll work on the unit tests right now and hopefully this will be ready.

I think that I'm going to remove the controller because it's making coverage smaller and I don't want to spend time on testing it, because you'll remove it

Reply:
OK